content/designer/reference/theming.rst

```diff
@@ -0,0 +1,587 @@
+view.
+
+By reading the source code, templates related to options are easily spottable.
+
+.. code-block:: xml
+    <we-button title="..."
+       data-name="..."
+       data-customize-website-views="website.template_header_default"
+       data-customize-website-variable="'...'"
+       data-img="..."/>
+
+.. code-block:: xml
+    <template id="..." inherit_id="..." name="..." active="True"/>
+    <template id="..." inherit_id="..." name="..." active="False"/>
+
+**Example**: Change menu items horizontal alignment
+
+.. code-block:: xml
+    :caption: ``/website_airproof/data/presets.xml``
+
+    <record id="website.template_header_default_align_center" model="ir.ui.view">
+       <field name="active" eval="True"/>
+    </record>
+
+The same logic can be used for others Odoo Apps as well.
+
+**Example**: E-commerce - Display Products Categories
+
+.. code-block:: xml
+
+    <record id="website_sale.products_categories" model="ir.ui.view">
+       <field name="active" eval="False"/>
+    </record>
+
+**Example**: Portal - Disable Language Selector
+
+.. code-block:: xml
+
+    <record id="portal.footer_language_selector" model="ir.ui.view">
+       <field name="active" eval="False"/>
+    </record>
+
+Assets
+======
+
+For this part, we will refer to the  assets_frontend bundle, located in the web module. This bundle
+specifies the list of assets loaded by the Website Builder, and our goal is to add our SCSS and JS
+files to it.
+
+Styles
+------
+The Odoo Website Builder and Bootstrap are great for defining the basic styles of your website.
+But to provide a unique design, you need to go a step further.
+For this, you can easily add any SCSS file to your theme.
+
+**Declaration**
+
+.. code-block:: python
+    :caption: ``/website_airproof/__manifest__.py``
+
+    'assets': {
+       'web.assets_frontend': [
+          'website_airproof/static/src/scss/theme.scss',
+       ],
+    },
+
+Feel free to reuse the variables in your `theme.scss` file (both the ones you put in your bootstrap
+file, and the ones used by odoo).
+
+**Example**
+
+.. code-block:: javascript
+    :caption: ``/website_airproof/static/src/scss/theme.scss``
+
+     blockquote {
+       border-radius: $rounded-pill;
+       color: o-color('o-color-3');
+       font-family: o-website-value('headings-font');
+     }
+
+Interactivity
+-------------
+
+Odoo supports three different kinds of javascript files:
+
+- `plain javascript files <https://www.odoo.com/documentation/15.0/developer/reference/frontend/javascript_modules.html#frontend-modules-plain-js>`_ (no module system),
+- `native javascript module <https://www.odoo.com/documentation/15.0/developer/reference/frontend/javascript_modules.html#frontend-modules-native-js>`_.
+- `Odoo modules <https://www.odoo.com/documentation/15.0/developer/reference/frontend/javascript_modules.html#frontend-modules-odoo-module>`_ (using a custom module system),
```

Comment:
never use absolute links to reference our doc.

Either use [custom anchors to target specific headings](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#custom-anchors)

or [documentation pages hyperlinks](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks).